Library/Homebrew/install.rb

```diff
@@ -10,13 +10,18 @@ module Install
     module_function
 
     def check_cpu
-      case Hardware::CPU.type
-      when :ppc
+      if OS.mac? && Hardware::CPU.ppc?
         abort <<~EOS
           Sorry, Homebrew does not support your computer's CPU architecture.
-          For PPC support, see:
+          For PowerPC Macs (PPC32/PPC64BE) support, see:
             #{Formatter.url("https://github.com/mistydemeo/tigerbrew")}
         EOS
+      elsif OS.linux? && Hardware::CPU.ppc64le?
```

Comment:
This should live in a new `Library/Homebrew/extend/os/linux/install.rb`. See the other files in that directory for the pattern.